SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialActorUtils.h

```diff
@@ -5,16 +5,18 @@
 #include "EngineClasses/SpatialNetConnection.h"
 
 #include "Components/SceneComponent.h"
-#include "Containers/Array.h"
 #include "Containers/UnrealString.h"
 #include "Engine/EngineTypes.h"
 #include "GameFramework/Actor.h"
 #include "GameFramework/Controller.h"
 #include "GameFramework/GameMode.h"
 #include "GameFramework/PlayerController.h"
-#include "GameplayDebuggerCategoryReplicator.h"
 #include "Math/Vector.h"
 
+#if WITH_UNREAL_DEVELOPER_TOOLS || (!UE_BUILD_SHIPPING && !UE_BUILD_TEST)
```

Comment:
I do not have a lot of context but why do we check if `!UE_BUILD_TEST` , I thought the issue is only in the Shipping build? Can we not just check `#if !UE_BUILD_SHIPPING` ?

Reply:
I checked the build file for the gameplay debugger: https://github.com/improbableio/UnrealEngine/blob/4.25-SpatialOSUnrealGDK-release/Engine/Source/Developer/GameplayDebugger/GameplayDebugger.Build.cs#L42

it looks like it's only realy supposed to be used for debug or development, so I used the same condition here